corehq/apps/data_dictionary/views.py

```diff
@@ -141,7 +143,11 @@ def update_case_property(request, domain):
                 errors.append(error)
 
     if errors:
-        return JsonResponse({"status": "failed", "errors": errors}, status=400)
+        if len(errors) == 1:
+            message = errors[0]
+        else:
+            message = mark_safe("<ul>" + "".join(["<li>{}</li>".format(format_html(e)) for e in errors]) + "</ul>")
+        return JsonResponse({"status": "failed", "message": message}, status=400)
```

Comment:
You sure this formatting shouldn't happen on the JS side?  Presumably this is a JSON response because it's called from some JS.

Reply:
I thought about it - I'd have updated `main.js` to accept a `messages` param instead and then do the formatting - but but didn't feel strongly about either way being better. Do you think it'd make more sense to do it that way?

Reply:
(offline this conversation)

Moved to front end in 97385221fa4